src/main/kotlin/io/rsocket/cli/Main.kt

```diff
@@ -152,9 +152,14 @@ class Main : Runnable {
       inputPublisher = LineInputPublishers(outputHandler)
     }
 
+    if (route != null) {
+      metadataFormat = "composite"
```

Comment:
@linux-china What are firm assumptions we can make when route is selected?

Reply:
it's a nice to make Spring route simple.  A question, I know some developers use cbor for data en/decoding with RSocket.  Could you comment  dataFormat assign(json by default now) when route is not null, and we can use route and cbor both. thanks. 

```
    if (route != null) {
      metadataFormat = "composite"
      // dataFormat = "json
 }
```

Reply:
Should I remove binary as the default?  Or check for it and then override to json?

I'd like to get it down to the minimal set of command line arguments to activate this.

Reply:
for binary, could we consider base64 text for cbor and other binary?

Reply:
binary as base64 text just for reference to implement CLI.  we can consider this feature later. Now json is good for CLI to Spring RSocket, and Spring RSocketStrategies supports to config multi encoders.

Reply:
OK, I'm only setting when unset.  Take a look.